src/perl/Permabit/BlockDevice/VDO.pm

```diff
@@ -578,7 +581,16 @@ sub installModule {
   my ($self) = assertNumArgs(1, @_);
   my $machineName = $self->getMachineName();
   my $moduleName = $self->getModuleName();
+
+  if ($self->{useUpstreamModule}) {
+    my $getVerCmd = "yum list $VDO_USER_MODNAME.`uname -m` | " .
+                    "awk '/^$VDO_USER_MODNAME/ {print \$2}'";
+    my @ver = split(/\./,
+		    runCommand($self->getMachineName(), $getVerCmd)->{stdout});
+    $self->setModuleVersion("$ver[0].$ver[1]");
+  }
```

Comment:
This code looks basically reasonable, but I think it ends up looking cleaner if this moves inside of getModuleVersion().